src/Core/Installer.php

```diff
@@ -243,7 +248,7 @@ public function checkPHP($phppath = null, $required = false)
 			$help .= EOL . EOL;
 			$tpl = Renderer::getMarkupTemplate('field_input.tpl');
 			$help .= Renderer::replaceMacros($tpl, [
-				'$field' => ['phpath', L10n::t('PHP executable path'), $phppath, L10n::t('Enter full path to php executable. You can leave this blank to continue the installation.')],
+				'$field' => ['config.php_path', L10n::t('PHP executable path'), $phppath, L10n::t('Enter full path to php executable. You can leave this blank to continue the installation.')],
```

Comment:
This isn't consistent with the input fields in `view/templates/install/*` which are named `config-php_path`. Is this intended? Am I comparing apples to oranges?

Reply:
I'll change it and leave a reminder that we mix backend classes with the "presentation layer" (code smell)

Thanks for the hint